modules/imgproc/src/spatialgradient.cpp

```diff
@@ -129,133 +129,130 @@ void spatialGradient( InputArray _src, OutputArray _dx, OutputArray _dy,
 
     int i_start = 0;
     int j_start = 0;
-#if CV_SIMD128
-    if(hasSIMD128())
+#if CV_SIMD
+    uchar *m_src;
+    short *n_dx, *n_dy;
```

Comment:
Do we need these variables outside of the for loop below?

Reply:
Moved variables inside loops